### ISSUE TITLE
[anim] Ender chest can not be animated

### ISSUE THREAD

**Issue by MaxiKevey** [sp614x/optifine#5416]
## Description of Issue
The ender chest can't be animated via custom animations. Other entities work fine.

.properties file looks like this:

x=56
w=56
h=96
y=76
to=textures/entity/chest/ender.png
from=./enderchest.png

## OptiFine Version
G7, G6, G5 and a few more

**Comment by sp614x** [comment]
The chest textures are packed in a texture atlas at runtime and can't be animated directly via texture animations.

For example:
![image](https://user-images.githubusercontent.com/16635441/110461058-463e3680-80cf-11eb-8273-fdf3ae5032d3.png)

**Comment by MaxiKevey** [comment]
Ah, that's a bummer. I guess there is no way around it then. Would be a cool feature though. Especially for the enderchest.

**Comment by MaxiKevey** [comment]
I just checked with MCPatcher and a custom animation seems to work on the chest textures. So there must be a way to get that to work, or did Mojang change something in an update that made this impossible?

**Comment by sp614x** [comment]
Up to 1.14 there was only one atlas texture for the blocks/items.
1.15 added atlas textures for chests, signs, banners, beds, paintings, etc.
The atlas textures would need a CTM-like functionality which can change single sprites in the atlas.

**Comment by sp614x** [comment]
Is the ".mcmeta" animation not working?

**Comment by MaxiKevey** [comment]
No, doesn't seem to work. Just stretches the texture across the model.

![2021-03-15_21 41 43](https://user-images.githubusercontent.com/28632262/111218815-86258200-85d7-11eb-8cec-ab5b2e573028.png)

**Comment by minthousebuilder** [comment]
It does work, however the file size is larger than is desirable, did you do the ender.png.mcmeta right?

**Comment by MaxiKevey** [comment]
Ah, yeah sorry you're right, there was a wrong symbol in the file. But still, like you said, the file size is unholy. So basically not an option.

**Comment by sp614x** [comment]
Currently the sprites of an atlas texture can only be animated with a `.mcmeta` animation.

**Comment by MaxiKevey** [comment]
Ah, okay alright. Thanks for the info. I changed the title to a feature request now, since it is not a bug. Would be a really useful thing to have. Especially to get the resource pack and atlas file size down.
